tasks/agent-win.yml

```
@@ -50,6 +50,15 @@
   with_items: "{{ datadog_checks|list }}"
   notify: restart datadog-agent-win
 
+- name: Ensure datadog-trace-agent and datadog-process-agent are not disabled
+  win_service:
+    name: "{{item}}"    
```

Made a pass with `ansible-lint`, it complained about this:

```suggestion
    name: "{{ item }}"
```